src/sass/base/_va.scss

```diff
@@ -1981,3 +1981,23 @@ pre[class*='language-']  {
   border-left: 2px solid $color-primary;
   padding-left: 1rem;
 }
+
+// Flexbox related utilities 
+.va-flex {
+  display: -webkit-flex;
```

Comment:
I don't really understand the webpack config format, but it doesn't look like it's in there. We should think about adding it later. I'll put in a ticket for it.